app/domain/authentication/authn_jwt/validate_input.rb

```diff
@@ -0,0 +1,27 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+
+    ValidateInput ||= CommandClass.new(
+      dependencies: {
+        jwt_validate_body: Authentication::AuthnJwt::ValidateRequestBody.new
```

Comment:
please call the param in the same name as the class:
```suggestion
        validate_request_body: Authentication::AuthnJwt::ValidateRequestBody.new
```